packages/@expo/cli/src/export/exportDomComponents.ts

```diff
@@ -157,15 +157,28 @@ export function updateDomComponentAssetsForMD5Naming({
   const htmlContent = files.get(htmlOutputName);
   assert(htmlContent);
   const htmlMd5 = crypto.createHash('md5').update(htmlContent.contents.toString()).digest('hex');
-  const hash = crypto.createHash('sha1').update(domComponentReference).digest('hex');
+  const hash = crypto.createHash('md5').update(domComponentReference).digest('hex');
   for (const artifact of nativeBundle.artifacts) {
     if (artifact.type !== 'js') {
       continue;
     }
     const assetEntity = files.get(artifact.filename);
     assert(assetEntity);
-    const regexp = new RegExp(`(['"])${hash}\\.html(['"])`, 'g');
-    assetEntity.contents = assetEntity.contents.toString().replace(regexp, `$1${htmlMd5}.html$2`);
+    if (Buffer.isBuffer(assetEntity.contents)) {
```

Comment:
I can't really tell what's going on here

Reply:
for hermes bytecode bundle, basically `assetEntity.contents.toString()` aka `Buffer.toString('utf8')` will break its binary content.